vllm/worker/worker_base.py

```diff
@@ -531,7 +532,10 @@ def adjust_rank(self, rank_mapping: Dict[int, int]) -> None:
     def update_environment_variables(self, envs_list: List[Dict[str,
                                                                 str]]) -> None:
         envs = envs_list[self.rpc_rank]
-        key = 'CUDA_VISIBLE_DEVICES'
+        if current_platform.is_xpu():
+            key = current_platform.device_control_env_var
```

Comment:
can just keep this?

Reply:
yes, it's kept when current platform is not xpu.